File: Python/1. Introduction/exercise2.py
#!/bin/python3

# Python If-Else "https://www.hackerrank.com/challenges/py-if-else/problem"


if __name__ == '__main__':
    n = int(input().strip())
    if n % 2 != 0 or (n % 2 == 0 and 6 < n < 21):
        print("Weird")
    elif n % 2 == 0 and (2 < n < 6 or n > 20):
        print("Not Weird")

File: Java/1. Introduction/exercise2.java
// Java Stdin and Stdout I "https://www.hackerrank.com/challenges/java-stdin-and-stdout-1/problem"

import java.util.Scanner;


public class Exercise2 {

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int a = scan.nextInt();
        int b = scan.nextInt();
        int c = scan.nextInt();

        System.out.println(a);
        System.out.println(b);
        System.out.println(c);

        // A more concise solution
        // while(scan.hasNextInt()){
        //     int a = scan.nextInt();
        //     System.out.println(a);
        // }

        scan.close();
    }
}
